examples/standardLayers/kde.m

```diff
@@ -0,0 +1,145 @@
+function [bandwidth,density,xmesh,cdf]=kde(data,n,MIN,MAX)

```

Comment:
This file is not used, is it required?

Reply:
It's a smoother for histograms - it may get used please keep

Reply:
okay, I will move this to utilities folder